pkg/rcmanager/ipinstance.go

```diff
@@ -0,0 +1,210 @@
+	klog.Infof("[remote cluster] Starting reconcile ipinstance from cluster %v, node name=%v", m.ClusterName, nodeName)
+	if len(nodeName) == 0 {
+		return nil
+	}
+	var (
+		node     *v1.Node
+		err      error
+		vtepName string
+	)
+	vtepName = utils.GenRemoteVtepName(m.ClusterName, nodeName)
+	node, err = m.NodeLister.Get(nodeName)
+	if err != nil {
+		if k8serror.IsNotFound(err) {
+			err = m.LocalClusterRamaClient.NetworkingV1().RemoteVteps().Delete(context.TODO(), vtepName, metav1.DeleteOptions{})
+			if k8serror.IsNotFound(err) {
+				return nil
+			}
+		}
+		return err
+	}
+
+	var (
+		vtepIP     = node.Annotations[constants.AnnotationNodeVtepIP]
+		vtepMac    = node.Annotations[constants.AnnotationNodeVtepMac]
+		newVtep    = false
+		remoteVtep *networkingv1.RemoteVtep
+	)
+
+	remoteVtep, err = m.RemoteVtepLister.Get(vtepName)
+	if err != nil {
+		if !k8serror.IsNotFound(err) {
+			return err
+		}
+		remoteVtep = utils.NewRemoteVtep(m.ClusterName, m.RemoteClusterUID, vtepIP, vtepMac, node.Name, nil)
+		newVtep = true
+	}
+
+	var (
+		curTime    = metav1.Now()
+		desired    []string
+		desiredSet *gset.StrSet
+		actual     []string
+		actualSet  *gset.StrSet
+		remove     *gset.StrSet
+		add        *gset.StrSet
+		// vtepIP or vtepMac changed
+		vtepChanged           bool
+		endpointILListChanged bool
+	)
+
+	desired, err = m.pickEndpointListFromNode(node)
+	if err != nil {
+		return err
+	}
+	actual = remoteVtep.Spec.EndpointIPList
+	desiredSet = gset.NewStrSetFrom(desired)
+	actualSet = gset.NewStrSetFrom(actual)
+	remove = actualSet.Diff(desiredSet)
+	add = desiredSet.Diff(actualSet)
+
+	remoteVtep = remoteVtep.DeepCopy()
+	remoteVtep.Spec.VtepIP = vtepIP
+	remoteVtep.Spec.VtepMAC = vtepMac
+	remoteVtep.Spec.EndpointIPList = desiredSet.Slice()
+	remoteVtep.Status.LastModifyTime = curTime
+
+	vtepChanged = vtepIP != "" && vtepMac != "" && (vtepIP != remoteVtep.Spec.VtepIP || vtepMac != remoteVtep.Spec.VtepMAC)
+	endpointILListChanged = remove.Size() != 0 || add.Size() != 0
+	if !newVtep && !endpointILListChanged && !vtepChanged {
+		return nil
+	}
+	if newVtep {
+		remoteVtep, err = m.LocalClusterRamaClient.NetworkingV1().RemoteVteps().Create(context.TODO(), remoteVtep, metav1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+	} else {
+		remoteVtep, err = m.LocalClusterRamaClient.NetworkingV1().RemoteVteps().Update(context.TODO(), remoteVtep, metav1.UpdateOptions{})
+		if err != nil {
+			return err
+		}
+	}
+	_, err = m.LocalClusterRamaClient.NetworkingV1().RemoteVteps().UpdateStatus(context.TODO(), remoteVtep, metav1.UpdateOptions{})
+	return err
+}
+
+func (m *Manager) pickEndpointListFromNode(node *v1.Node) ([]string, error) {
+	instances, err := m.IPLister.List(labels.SelectorFromSet(labels.Set{constants.LabelNode: node.Name}))
+	if err != nil {
+		return nil, err
+	}
+	desired := utils.PickUsingIPList(instances)
+	nodeLocalVxlanipStringList := strings.Split(node.Annotations[constants.AnnotationNodeLocalVxlanIPList], ",")
```

Comment:
Considering it's a separate logic for daemon to config iptables rules for "node local vxlan ip", simply adding node local vxlan list into endpoint list seems not a good solution, because daemon cannot find out which are node ip addresses in endpoint list.

Can we keep the node local vxlan ip list an independent ip list? Just like copy the `node.Annotations[constants.AnnotationNodeLocalVxlanIPList]` into remote vtep CR's annotations.